tabular/src/autogluon/tabular/configs/hyperparameter_configs.py

```diff
@@ -79,23 +79,9 @@
         ],
         'CAT': {},
         'XGB': {},
-        # 'FASTAI': {},  # FIXME: FASTAI is extremely slow if trained alongside TextNN due to forkserver.
-        'RF': [
-            {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
-        ],
-        'XT': [
-            {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
-        ],
-        'KNN': [
-            {'weights': 'uniform', 'ag_args': {'name_suffix': 'Unif'}},
-            {'weights': 'distance', 'ag_args': {'name_suffix': 'Dist'}},
-        ],
-        'TEXT_NN_V1': ['ELECTRA_BASE'],
+        # 'FASTAI': {}, FastAI gets killed if the dataset is large (400K rows).
         'custom': ['GBM'],
+        'AG_TEXT_NN': ['medium_quality'],
```

Comment:
If user specifies `presets` in TabularPredictor.fit(), will these also affect the creation of AG_TEXT_NN?  I think it should, eg. `best_quality` can use larger AG_TEXT_NN than the default.

Reply:
This could be done in future PR though, if you mark as TODO

Reply:
Don't do this change now, just have as TODO.

Reply:
Anyway, I changed it.

Reply:
Oh, I misread the issue. Yes, this is marked as a TODO.